package.json

```diff
@@ -30,7 +30,7 @@
   },
   "scripts": {
     "start": "HTTPS=true react-scripts start",
-    "build": "react-scripts build && cp -a ./build/. ./cordova-app/www/",
+    "build": "react-scripts build && cp -a ./build/. ./cordova-app/www/ && cp .gitignore gitignore",
```

Comment:
Why need to copy to a new gitignore folder?

Reply:
it is not a folder: it is a copy of .gitignore. it is a work around as .gitignore is not published.